kpi/serializers/current_user.py

```diff
@@ -75,18 +75,47 @@ def get_git_rev(self, obj):
     def to_representation(self, obj):
         if obj.is_anonymous:
             return {'message': 'user is not logged in'}
+
         rep = super().to_representation(obj)
-        if not rep['extra_details']:
+        if not rep['extra_details'] or not isinstance(
+            rep['extra_details'], dict
+        ):
```

Comment:
We already add a discussion with @joshuaberetta in other PR about this format in `if` statement. 
My eyes always try to find the opening parenthesis next to `if` and make my brain think there is an error. 

What about this instead?:
```python
if (
  not rep['extra_details'] 
  or not isinstance(rep['extra_details'], dict)
):
```

It has also the advantage to put the operator on another line.

There is an internal discussion in Zulip: https://chat.kobotoolbox.org/#narrow/stream/4-KoBo-Dev/topic/Python.20coding.20style/near/5423

Reply:
this was formatted by Black :shrug:
i agree that your suggestion is nicer, except that you're indenting by 2 spaces (?)

Reply:
applied the suggested formatting but with 4-space indentation

Reply:
Sorry, I did it quick. 4-space it should have been ;-)